mlflow/tracing/enablement.py

```diff
@@ -0,0 +1,141 @@
+"""
+Trace enablement functionality for MLflow to enable tracing to Databricks Storage.
+"""
+
+import logging
+
+from mlflow.entities.trace_location import UCSchemaLocation
+from mlflow.exceptions import MlflowException
+from mlflow.tracing.client import TracingClient
+from mlflow.tracking._tracking_service.utils import get_tracking_uri
+from mlflow.tracking.fluent import _get_experiment_id
+from mlflow.utils.annotations import experimental
+from mlflow.utils.uri import is_databricks_uri
+
+_logger = logging.getLogger(__name__)
+
+
+@experimental(version="3.5.0")
+def set_experiment_storage_location(
```

Comment:
```suggestion
def set_experiment_trace_location(
```

nit: do you think it's better to have "trace" in the name?